### ISSUE TITLE
WebWallet refresh/report/sync issues

### ISSUE THREAD

**Issue by jpenner64** [duino-coin/duino-coin#1171]
Symptom: Wallet dashboard Page periodically reflects miner status, then drops out despite miners/rig actively hashing and constantly reporting shares. When it drops it reports as if there is a connection issue with the miners/rig, however when it comes back it is clear that ongoing shares are being logged during this time. Also, reporting earnings/DUCO freezes and is stuck in calculating...

Attempt to isolate: Miners/rig restarted to ensure not end user issue. Access to and processing continues with host OK, but wallet not refreshing properly/accurately

Issue observed 12/14/2021 20:30-21:30 PST 

As of 21:50 it looks like it is reporting miners and calculating again. possible delay/loss on DUCO earnings as I have not seen it catch up to expected amount. 

So, might there have been an issue with the site, delay in reporting to wallet, maintenance, etc....?

**Describe the bug**

**To Reproduce**
Not known to be reproducible, but impact is seen here:
1. Go to https://wallet.duinocoin.com/ dashboard and under account and "Your Miners" sections

**Expected behavior**
Consistent and periodic reporting of Account earnings, and miner status

**Screenshots/photos**
Sorry, missed doing a screen shot before issue seemed to recover

**System (please complete the following information):**
 - OS: Windows 10
 - Software NondMCU ESP8266
 - Version 2.7

**Additional context**
Best guess a backend issue/delay from recording server to wallet dashboard as a result of outage or maintenance

**Comment by jpenner64** [comment]
Update: 12/15/2021 07:35 PST
Problem persists and it appear issues at Duino-Coin side with severs
I did notice there is a "Server Status" view which I checked yesterday, and assumed that the webWallet-API being down on 14 December was the cause. As of now, Server Status page times out, or is slow in loading. Currently, page shows but loading stats is taking a long time. Looks like a few servers are down....Including:

- webWallet-API - Down
- BeyondPool - Down
- NullPool - Down
